SUBMODULE_VERSIONS

```diff
@@ -5,7 +5,7 @@ daff5fead3fbe22c6fc58310ca3f49caf117f185 third_party/benchmark
 a5d9d0f7d368054fd1691aedf1db4116efcc233e third_party/flatbuffers
 4fb0ff7069bd88ee85902f4d0bb62794e5f6d021 third_party/flatcc
 f2fb48c3b3d79a75a88a99fba6576b25d42ec528 third_party/googletest
-6154c4115cd4b78d0171892aac21e340e72e32bd third_party/llvm-project
+c89447b65984c97145f63be21e42cfa98da60dd2 third_party/llvm-project
```

Comment:
Hey @GMNGeoffrey were there Windows build failures for TF/LLVM somewhere in this range? I see some chat messages about that, wondering if they'll affect us. If so, those of us building on Windows can just not sync past this :P

Reply:
There were. Someone broke building LLVM without threads enabled. But we also think the bug that forced us to disable threads on windows has been fixed and just wasn't marked as such.